webapp/src/components/StatusCheck.tsx

```diff
@@ -1,14 +1,44 @@
-import { Box, Typography } from "@mui/material";
+import { Box, capitalize, Stack, Typography } from "@mui/material";
 import noData from "assets/launch.svg";
 import React from "react";
 import { useParams } from "react-router-dom";
 import { getStatusEndpoint } from "services/api";
 import Loading from "./Loading";
+import Grid from "@mui/material/Grid";
+import Paper from "@mui/material/Paper";
+import CheckIcon from "./Icons/Check";
+import ErrorIcon from "@mui/icons-material/Error";
+import PendingOutlinedIcon from "@mui/icons-material/PendingOutlined";
 
 type Props = {
   children: React.ReactNode;
 };
 
+interface StringByString {
+  [key: string]: any;
+}
+
+const STATUS_ICON_MAPPING: StringByString = {
+  finished: <CheckIcon />,
+  not_started: <CheckIcon />,
+  pending: <PendingOutlinedIcon />,
+  error: <ErrorIcon />,
+  lost: <ErrorIcon />,
+};
+
+const render_cell = (task_name: string, task_status: string) => (
+  <Grid item xs={1.5}>
+    <Paper variant="outlined">
+      <Box display="flex" alignItems="center" gap={1}>
+        <Typography>
+          {`${task_name.split("_").map(capitalize).join(" ")}  `}
+          {STATUS_ICON_MAPPING[task_status]}
+        </Typography>
```

Comment:
I think I would capitalize only the first word, and not all of them. Some letters are obviously wrong, even as title, like `By` in `Metrics By Filter Train`. Also, the icon should be outside of the `Typography` for the `Box`'s `alignItems="center"` and `gap={1}` to work. I would put the icon to the left of the `Typography` so that all icons are aligned.
```suggestion
        {STATUS_ICON_MAPPING[task_status]}
        <Typography>{capitalize(task_name).replace(/_/g, " ")}</Typography>
```

Reply:
See comment https://github.com/ServiceNow/azimuth/pull/190/files#r942810300 first as it cover this too.